src/products/views/ProductUpdate/handlers/useProductUpdateHandler.ts

```diff
@@ -83,117 +79,126 @@ export function useProductUpdateHandler(
 
   const [updateMetadata] = useUpdateMetadataMutation({});
   const [updatePrivateMetadata] = useUpdatePrivateMetadataMutation({});
-  const [updateStocks] = useVariantDatagridStockUpdateMutation({});
-  const [updateVariant] = useVariantDatagridUpdateMutation();
+  const [updateVariants] = useProductVariantBulkUpdateMutation();
   const [createVariants] = useProductVariantBulkCreateMutation();
   const [deleteVariants] = useProductVariantBulkDeleteMutation();
 
   const [uploadFile] = useFileUploadMutation();
 
   const [updateProduct, updateProductOpts] = useProductUpdateMutation();
-  const [
-    updateChannels,
-    updateChannelsOpts,
-  ] = useProductChannelListingUpdateMutation({
-    onCompleted: data => {
-      if (!!data.productChannelListingUpdate.errors.length) {
-        data.productChannelListingUpdate.errors.forEach(error =>
-          notify({
-            status: "error",
-            text: getProductErrorMessage(error, intl),
-          }),
-        );
-      }
-    },
-  });
-
-  const [
-    updateVariantChannels,
-  ] = useVariantDatagridChannelListingUpdateMutation();
+  const [updateChannels, updateChannelsOpts] =
+    useProductChannelListingUpdateMutation({
+      onCompleted: data => {
+        if (!!data.productChannelListingUpdate.errors.length) {
+          data.productChannelListingUpdate.errors.forEach(error =>
+            notify({
+              status: "error",
+              text: getProductErrorMessage(error, intl),
+            }),
+          );
+        }
+      },
+    });
 
   const [deleteAttributeValue] = useAttributeValueDeleteMutation();
 
   const sendMutations = async (
```

Comment:
suggestion: Don't you thing sending all mutations deserves separate hook logic? It can expose some sort of "save" function with some success/error state